testing/requests.md

```diff
@@ -47,6 +47,43 @@ $this->following_redirects()
      ->assertSee( 'Success!' );
 ```
 
+### Request Cookies
+
+Requests can have cookies included with them:
+
+```php
+$this
+  ->with_cookie( 'session', 'cookie-value' );
+  ->get( '/endpoint' );
+
+// Pass multiple cookies.
+$this->
+  ->with_cookies( [ ... ] )
```

Comment:
what would the items in the array look like? An example might be good, since the single is just two separate params.

Reply:
Good point!